.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -1,102 +1,82 @@
 name: Bug Report
 description: File a bug report
 title: '[Bug]: '
-labels: ['Bug', 'Community backlog']
+labels: ['Bug', 'Community backlog', 'High priority']
+projects: ONS Design System project
 assignees: []
 body:
     - type: markdown
       attributes:
           value: |
               Thank you for taking the time to fill out this bug report
     - type: textarea
-      id: what-happened
+      id: describe-bug
       attributes:
-          label: What happened?
+          label: Describe the bug
+          description: 'If applicable, add screenshot(s) or examples of the bug/issue'
       validations:
           required: true
     - type: textarea
-      id: expected-outcome
+      id: expected-behaviour
       attributes:
-          label: What was the expected outcome?
-      validations:
-          required: true
-    - type: textarea
-      id: impact
-      attributes:
-          label: How is this problem impacting your service?
+          label: Expected behaviour / outcome
+          description: 'If applicable, add screenshot(s) or examples of the expected behaviour'
```

Comment:
"if applicable add screenshot(s) or examples of the bug/issue"
This is the wording on the card